e2e/cypress/integration/onboarding/next_steps_tips_spec.js

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @onboarding
+
+describe('Onboarding - Next Steps Tips', () => {
```

Comment:
Same comment as above (similar test for cloud).
I've added a new test case in Zephyr for this. Please see [MM-T4146](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T4146). There are two steps -- one for admin and the other one for regular user.